vshard/replicaset.lua

```diff
@@ -1033,6 +1033,9 @@ local replica_mt = {
         safe_uri = function(replica)
             local uri = luri.parse(replica.uri)
             uri.password = nil
+            if uri.ipv6 then
+                uri.host = '[' .. uri.host .. ']'
```

Comment:
I don't understand, why this change is needed. `safe_uri` function is used only to show info in `vshard.storage/router.info` and to print info to logs, is this what you need?

1. Why can't uri be passed with brackets already to vshard config, if it's ipv6?
2. You probably wanted to change `replicaset.buildall` function, not `safe_uri`

Reply:
ipv6 addr in `replica.uri` passing with brackets, but after `luri.format(uri)` address will be something like `::1:3301`.

Without this changes I got `Failed to bootstrap vshard: [string "local function func(...) local cartridge = re..."]:1: ....rocks/share/tarantool/cartridge/roles/vshard-router.lua:186: Bad argument #1 to errors.netbox_eval (net.box connection expected, got nil)` during `cartridge replicasets setup --bootstrap-vshard` operation. As I understand this happens because of replicaset.master.uri address without brackets.

In instance config file (instances.yml file) we set address with brackets.

Reply:
Could you please provide any sort of test.

Reply:
Done

Reply:
Plese take a look for this MRs https://github.com/tarantool/membership/pull/55, https://github.com/tarantool/cartridge/pull/2166